README.md

```diff
@@ -88,6 +88,7 @@ No modules.
 | <a name="input_filter_pattern"></a> [filter\_pattern](#input\_filter\_pattern) | The filter pattern to use. For more information, see [Filter and Pattern Syntax](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html)" | `string` | `""` | no |
 | <a name="input_iam_name_prefix"></a> [iam\_name\_prefix](#input\_iam\_name\_prefix) | Prefix used for all created IAM roles and policies | `string` | `"observe-logs-subscription"` | no |
 | <a name="input_iam_role_arn"></a> [iam\_role\_arn](#input\_iam\_role\_arn) | ARN of IAM role to use for Cloudwatch Logs subscription.<br>If this is not specified, then an IAM role is created. | `string` | `""` | no |
+| <a name="input_ignore_delete_errors"></a> [ignore\_delete\_errors](#input\_ignore\_delete\_errors) | Ignore CloudFormation stack errors from deletion events.<br><br>Setting this to true means that leftover Subscription Filters could remain. | `bool` | `false` | no |
```

Comment:
docs generator needs to be re-run to update the description here

Reply:
done